### PR TITLE
Update krix.json

### PR DESCRIPTION
**Pull request by Krix12**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/krix.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "email": "kristjan.kocevar89@gmail.com"
       },
     "record": {
-        "A": ["35.186.245.55"],
+        "CNAME": "krix12.github.io",
```
Comment (mtgsquad) [inline domains/krix.json:8]: MX & CNAME cannot be used together, please consider switching to a redirect or an A record.

**Review by mtgsquad — DISMISSED** [review]

**Review by MaskDuck — APPROVED** [review]

File: domains/krix.json [hunk 2]
```diff
@@ -5,11 +5,6 @@
         "email": "kristjan.kocevar89@gmail.com"
       },
     "record": {
-        "A": ["35.186.245.55"],
-        "MX": [
-            "mx1.improvmx.com",
-            "mx2.improvmx.com"
-        ],
-        "TXT": "v=spf1 include:spf.improvmx.com ~all"
+        "CNAME": "krix12.github.io"
```
Comment (MaskDuck) [inline domains/krix.json:8]: i think i've got 404?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/krix.json (hunk 2 above)
Comment (Krix12) [reply domains/krix.json:8]: Should I then replace it with https://krix12.github.io/Krix-portfolio/?

Comment (Krix12) [reply domains/krix.json:8]: Fixed now

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**